spec/java/util/Map.lsl

```diff
@@ -52,3 +56,13 @@ import java/lang/Object;
 
 // global aliases and type overrides
 
+
+@GenerateMe
+@implements("java.io.Serializable")
+@implements("java.util.Map.Entry")
+@public type SimpleEntry
+    is java.util.SimpleEntry
```

Comment:
@dog-m Yes, you are right; At first I decided to create new type because looked at "Map.entry" static method; It was impossible to create node with null key and value with this approach; And I have forgotten that SimpleEntry allow this.

Fixed - This type was deleted;

Reply:
Fixed;